Cargo.toml

```diff
@@ -69,3 +69,6 @@ commcid = { path = "./utils/commcid" }
 forest_vm = { path = "./vm" }
 forest_runtime = { path = "./vm/runtime" }
 forest_message = { path = "./vm/message" }
+
+[profile.dev]
+debug = 0
```

Comment:
Does this improve build times?

Reply:
Yes, unless you need debug symbols.

I found this article helpful: https://matklad.github.io/2021/09/04/fast-rust-builds.html